tests/e2e/pageobjects/ide/OpenEditors.ts

```diff
@@ -25,7 +25,7 @@ export class OpenEditors {
     private static readonly OPEN_EDITORS_CLOSE_ALL_BUTTON_LOCATOR: By = By.id('navigator.close.all.editors.toolbar');
 
     constructor(
-        @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper
+        @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper,
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.